src/cockpit/channels/dbus.py

```diff
@@ -225,18 +223,28 @@ def do_open(self, options):
         self.tasks = set()
 
         bus = options.get('bus')
+        address = options.get('address')
 
-        if bus == 'internal':
+        if bus == 'none':
+            if address is None or address == "internal":
+                logger.debug('get internal bus for %s', self.name)
+                self.bus = self.router.internal_bus.client
+            else:
```

Comment:
does anyone actually do that?

Reply:
Good question. The C bridge supports it, but...

Reply:
I'd prefer to drop this part.  There's no sense in having two ways to do the same thing, particularly if we don't know that it's actually required.

Reply:
Agreed.